ci-operator/config/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master.yaml

```diff
@@ -7,26 +11,38 @@ base_images:
     name: release
     namespace: openshift
     tag: rhel-9-release-golang-1.22-openshift-4.17
+  operator-sdk:
+    name: "4.18"
+    namespace: origin
+    tag: operator-sdk
 build_root:
   from_repository: true
+  use_build_cache: true
 images:
-- inputs:
-    ocp_4.17_base-rhel9:
-      as:
-      - registry.ci.openshift.org/ocp/4.17:base-rhel9
-    openshift_release_rhel-9-release-golang-1.22-openshift-4.17:
-      as:
-      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.22-openshift-4.17
-  to: vertical-pod-autoscaler-operator
-- dockerfile_path: images/operator-registry/Dockerfile.registry.ci
+- build_args:
+  - name: TARGETOS
+    value: linux
+  - name: TARGETARCH
+    value: amd64
+  dockerfile_path: Dockerfile
   inputs:
     ocp_4.17_base-rhel9:
       as:
       - registry.ci.openshift.org/ocp/4.17:base-rhel9
     openshift_release_rhel-9-release-golang-1.22-openshift-4.17:
       as:
       - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.22-openshift-4.17
```

Comment:
We're kind of mixing and matching 4.17 and 4.18 here, I assume we will true up the 4.17/4.18 business once https://github.com/openshift/vertical-pod-autoscaler-operator/pull/169 merges?

Reply:
Yeah, 4.18 got cut in the middle of this PR, so I think we can have followup PRs on both repos to update the images to 4.18 on both after the whole migration is merged.